ibis/backends/pyspark/compiler.py

```diff
@@ -1772,6 +1772,26 @@ def compile_not_null(t, expr, scope, timecontext, **kwargs):
     return ~F.isnull(col) & ~F.isnan(col)
 
 
+@compiles(ops.DropNa)
+def compile_dropna_table(t, expr, scope, timecontext, **kwargs):
+    op = expr.op()
+    table = t.translate(op.table, scope, timecontext)
+    subset = [col.get_name() for col in op.subset] if op.subset else None
+    return table.dropna(how=op.how, subset=subset)
```

Comment:
This documentation here says "omitting rows with null values", but it works with "Na" values too?

https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.sql.DataFrame.dropna.html

Reply:
Well it seems that `pyspark.sql.DataFrame.dropna` does handle both `Null` and `Na`. 

`pyspark.sql.functions.isnan` only handles `Na`

`pyspark.sql.functions.isnull` only handles `Null`

Kind of confusing behavior but I think what you are doing here is correct